_includes/sidebar-data-cockroachcloud.json

```diff
@@ -252,15 +252,15 @@
       	  ]
       	},
         {
```

Comment:
Also need to add the changes in page to the `_redirects` file (https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#file-naming), so that anyone who tries to get to the former Run Bulk Operations page will be redirected to the Take and Restore Customer-Owned Backups.